### ISSUE TITLE
Following Bug

### ISSUE THREAD

**Issue by krixano** [HelloZeroNet/ZeroMe#99]
When I click Follow on someone's profile and go back to home page, it shows their posts in the followed users tab. However, refreshing the page will make those posts dissapear until I click Follow again on their profile.

**Comment by krixano** [comment]
It should be noted that the very first person you follow doesn't have this problem. But all others do.

**Comment by HelloZeroNet** [comment]
I have tried to reproduce it:
 - Went to Everyone feed
 - Clicked on one user's name (Glightstar)
 - Clicked on big `+ Follow` button
 - Went back to Followed users feed (Glightstar was there)
 - Refreshed
 - Glightstar posts was still there

Am I missing something?

What browser are you using? When you click on follow button does the publishing message appears?

**Comment by krixano** [comment]
Nope. That's exactly what I did. The only difference is that the posts *do not* show up. It happens in all browsers.

**Comment by krixano** [comment]
I looked at my data.json file in my hub (kaffiene hub; `\data\1oranGeS2xsKZ4jVsu9SVttzgkYXu4k9v\data\users\12gAes6NzDS9E2q6Q1UXrpUdbPS6nvuBPu`) and noticed that I have a lot of duplicates.

![image](https://user-images.githubusercontent.com/4098042/33530182-7a454764-d841-11e7-95a6-114dac645f2e.png)

**Comment by krixano** [comment]
Another thing: After removing all followers from my data.json file, Glightstar still shows up as followed.

**Comment by krixano** [comment]
I have found another weird thing. I looked at the ZeroMe sql database in an sql browser. In the json table, I searched for my name from ther "cert_user_id" column. If you look below, all sites show the hub as "12h51..." (kaffiene hub) *except* for "1White24..." (White Hub), which shows the hub as "1White24".

![image](https://user-images.githubusercontent.com/4098042/33530378-3bab43f2-d844-11e7-84b2-2c1889fd2576.png)

Upon further investigation, I see that I have two followers, one of which is Glightstar in the data file under White Hub:

![image](https://user-images.githubusercontent.com/4098042/33530394-772b1538-d844-11e7-901b-a9901337f407.png)

So my computer was using the followers from White Hub instead of Kaffiene Hub.

**Comment by HelloZeroNet** [comment]
Thanks for the investigation. Normally there should be no more than one profile for one user. To avoid this I added a confirmation on new profile creation when the user already have one: https://github.com/HelloZeroNet/ZeroMe/commit/eba38e3466996a973935291d677c80e24765d60d
And also added a hub selection to the 3dot menu if more than one profile found for the same user: https://github.com/HelloZeroNet/ZeroMe/commit/614851064efec30d418d161c14b9fe9dbb429764

I hope this solves the problems

**Comment by 0polar** [comment]
http://127.0.0.1:43110/1QHbf72f4vdx3QAxN1LTN9KP19J428Q85R
